### ISSUE TITLE
Allow changing non-IE batch size in markerclustererplus

### ISSUE THREAD

**Issue by doom777** [googlemaps/v3-utility-library#497]
Copy of #136 

Currently you can set `batchSizeIE`, but not `batchSize`. You should do both.

**Comment by garylittleRLP** [comment]
This can already be done.

**Comment by doom777** [comment]
No, there is no getter or setter.

**Comment by garylittleRLP** [comment]
When you create the MarkerClusterer you can pass in a bunch of options. One is batchSize and another is batchSizeIE.

**Comment by doom777** [comment]
Correct, but if you want to change a MarkerClusterer's batchSizeIE you can via `setBatchSizeIE` method, but there is no corresponding method for `setBatchSize`

**Comment by garylittleRLP** [comment]
OK, I get it now. Yes, that's missing and I can add it in.

**Comment by doom777** [comment]
Not fixed yet.

**Comment by doom777** [comment]
NOt fixed yet.
